src/util.h

```diff
@@ -13,4 +13,8 @@
 **/
 #define ARRAY_SIZE(arr)     ( sizeof(arr) / sizeof((arr)[0]) )
 
+
+static inline Int AlwaysYes( Obj obj) { return 1; }
+static inline Int AlwaysNo( Obj obj) { return 0; }
+
```

Comment:
If we stick with the formatting of functions, making these functions 4 lines long, they should probably go into `util.c`.

I am just not sure whether these (essentially) one-liners should not stay one- or two-liners and stay in the header file (is this achievable with clang format? One of the reasons we might want to go with 4 lines is consistency and clang-format playing ball)

That said, I am not going to fight hard for either option.